src/devices/Pn532/OperatingMode.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Pn532
+{
+    /// <summary>
+    /// The operation mode
+    /// 0b0000_0011 is reserved for future usage
+    /// </summary>
+    public enum OperatingMode
+    {
+        HSU = 0b0000_0000,
```

Comment:
HSU? Is that SerialPort? (if so, could you rename?)

Reply:
It is a bit different. HSU is High-Speed UART. But HSU is usually communicating at higher speed than standard Serial Port.

Reply:
Perhaps expand the name to `HighSpeedUart`